src/applications/bmqtool/m_bmqtool_interactive.cpp

```diff
@@ -474,11 +578,8 @@ void Interactive::processCommand(const PostCommand& command, bool hasMPs)
 
         if (hasMPs) {
             d_session_p->loadMessageProperties(&properties);
-
-            for (size_t j = 0; j < command.messageProperties().size(); ++j) {
```

Comment:
Not sure I understand what is the purpose of this `for` cycle. Inside InputUtil::populateProperties() there is also `for` cycle which is used to process properties. Removed it.